use/2016-12-13_MEM_outlier.Rmd

```diff
@@ -0,0 +1,212 @@
+---
+title: Spatial detection of outlier loci with Moran eigenvector maps (MEM)
+---
+
+## Introduction
+
+The purpose of this vignette is to illustrate a new outlier detection method that uses genetic data and spatial coordinates to detect loci potentially under selection (Wagner et al. 2016). This method, which we call **Moran spectral outlier detection (MSOD)**, uses the power spectrum of **Moran eigenvector maps (MEM)** to quantify how variation in the frequency of an allele is distributed across a range of spatial scales defined by MEM spatial eigenvectors. MSOD can be used on individual or population based data sets.
+
+If environmental predictors are available, MSOD can be followed by a second step that uses **Moran spectral randomization (MSR)** to test associations between outlier loci and environmental predictors, while accounting for spatial autocorrelation.
+
+## Assumptions
+
+MSOD relies on an assumption of normality under the null hypothesis that the distributions of loci are shaped by the same process (gene flow). Deviations from normality are likely under non-equilibrium and isolation by distance scenarios, which may increase false positive rates. MSR relies on stationarity assumptions, which means that p-values may be inflated if both variables have a linear trend. See Wagner et al. 2016 for more discussion of these assumptions, and how they impact outlier detection in a variety of simulation scenarios. 
+
+
+## Data & packages
+
+In this vignette, we'll walk through an MSOD/MSR example using part of an individual-based simulation data set from Forester et al. 2016. In this case, the genetic data are input as allele counts (i.e. 0/1/2) for each locus. For population-based data, you can input the genetic data as allele frequencies within demes.
+
+Begin by installing the necessary packages, if you don't already have them:
+
+```{r set global_options, echo=F}
+knitr::opts_knit$set(root.dir = 'F:/12-MEMoutliers/tutorial/')
+```
+```{r packages global_options, include=TRUE, results="hide", message=FALSE, warning=FALSE}
+# install.packages(c("spdep","adespatial"), dependencies=TRUE)
+
+# Load packages
+# -------------
+require(spdep)
+require(adespatial)

```

Comment:
Style comment: `library("packageName")` is preferred over `require(packageName)`, but not necessary to change